task.go

```diff
@@ -118,6 +118,8 @@ type Task interface {
 	Update(context.Context, ...UpdateTaskOpts) error
 	// LoadProcess loads a previously created exec'd process
 	LoadProcess(context.Context, string, IOAttach) (Process, error)
+	// Metrics returns task metrics for runtime specific metrics
```

Comment:
It's not very obvious what the type of the returned value. I know we want to make it generic to support different platforms.

Can we add some examples in the comment? E.g.
```
E.g. for linux runtime, it will be `containerd/cgroups.Stats`.
```

Reply:
done